articles/azure-monitor/insights/container-insights-faq.md

```diff
@@ -9,6 +9,46 @@ ms.date: 10/15/2019
 
 This Microsoft FAQ is a list of commonly asked questions about Azure Monitor for containers. If you have any additional questions about the solution, go to the [discussion forum](https://feedback.azure.com/forums/34192--general-feedback) and post your questions. When a question is frequently asked, we add it to this article so that it can be found quickly and easily.
 
+## I dont see 'Image' and 'Name' fields populated for my logs in ContainerLog table

```

Comment:
rename H2 to - I don't see Image and Name property values populated when I query the ContainerLog table.